<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Training</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>300</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_machine_learning">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>841</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab_ModelBuilding">
    <attribute name="title">
     <string>Segment Images</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_SelectModel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>821</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Select Model</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_segmentation_model_file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>700</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>You must select the model that you want to use for segmenting the images.</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Select_Model">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>93</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_Select_Images_Folder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>821</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Select Folder with Images </string>
     </property>
     <widget class="QLineEdit" name="lineEdit_segmentation_images_folder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>700</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>You must select a folder containing the images that you want to segment.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Select_Images_Folder">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>93</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>400</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="checkBox_saveModelMasks">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Save Model Masks</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_copyOriginalModelImage">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>200</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Copy Original Model Image</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_Segment">
     <property name="geometry">
      <rect>
       <x>689</x>
       <y>30</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Segment Images</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Model Building</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_SelectTrainingImageFolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>821</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Training Images Folder</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Training_Images_Folder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>700</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>This field is pre-populated and cannot be changed at this time.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Select_Training_Images_Folder">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>93</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_SelectAnnotationFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>821</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>Annotation File (COCO)</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_Annotation_File">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>700</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="toolTip">
       <string>This field is pre-populated and cannot be changed at this time.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Select_Annotation_File">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>93</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_Tune_Model">
     <property name="geometry">
      <rect>
       <x>357</x>
       <y>175</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tune Model</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
